scripts/eqt_env.yml

```diff
@@ -1,9 +0,0 @@
-# This creates an environment with all of the prerequisites for eqt
```

Comment:
please commit the recipe folder and yml file

Reply:
Added recipe folder and yml file to branch